guides/05_chatbots/01_creating-a-chatbot-fast.md

```diff
@@ -136,6 +136,25 @@ gr.ChatInterface(
 
 The placeholder appears vertically and horizontally centered in the chatbot.
 
+If you would like to attach event listeners to your custom chatbot, wrap the chatbot as well as the `gr.ChatInterface` inside of a `gr.Blocks` like this:
```

Comment:
feel like the "creating a chatbot fast" guide should be quick and sweet, this particular usecase doesn't seem common enough to be here.

Reply:
That's fair, this guide is getting quite bloated. It would be better to move this over to the docs page for `gr.ChatInterface`. Let me do that